apps/coscrad-frontend/src/components/notes/note-index.presenter.tsx

```diff
@@ -1,18 +1,92 @@
-import { INoteViewModel } from '@coscrad/api-interfaces';
-import { Link } from 'react-router-dom';
+import {
+    EdgeConnectionMemberRole,
+    EdgeConnectionType,
+    IEdgeConnectionMember,
+    INoteViewModel,
+} from '@coscrad/api-interfaces';
+import { HeadingLabel, IndexTable } from '../../utils/generic-components/presenters/tables';
+import { CellRenderersDefinition } from '../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
 import { HasData } from '../higher-order-components';
+import { renderAggregateIdCell } from '../resources/utils/render-aggregate-id-cell';
 
-export const NoteIndexPresenter = ({ data: notes }: HasData<INoteViewModel[]>): JSX.Element => (
-    <div>
-        {notes.map((note) => (
-            <div key={note.id} data-testid={note.id}>
-                <h1>
-                    note: <Link to={note.id}>{note.id}</Link>
-                </h1>
-                <p>{note.note}</p>
-                <h2>Full Props (stringified):</h2>
-                <p>{JSON.stringify(note)}</p>
+interface DisplayConnectedResourcesInfoProps {
+    resourceInfos: IEdgeConnectionMember[];
+}
+
+const DisplayConnectedResourcesInfo = ({
+    resourceInfos,
+}: DisplayConnectedResourcesInfoProps): JSX.Element => {
+    if (resourceInfos.length === 1) {
+        const {
+            compositeIdentifier: { type, id },
+        } = resourceInfos[0];
+
+        return (
+            <div>
+                A note about {type}/{id}
             </div>
-        ))}
-    </div>
-);
+        );
+    }
+
+    const {
+        compositeIdentifier: { type: type0, id: id0 },
+        role: role0,
+    } = resourceInfos[0];
+
+    const {
+        compositeIdentifier: { type: type1, id: id1 },
+    } = resourceInfos[1];
+
+    const stringCompositeId0 = `${type0}/${id0}`;
+
+    const stringCompositeId1 = `${type1}/${id1}`;
+
+    const fromMessage =
```

Comment:
I can see that you're determining the role of the resources here.

Reply:
Yes.